Commands/SitesMassWordPressUpdateCommand.php

```diff
@@ -364,10 +366,35 @@ private function update($update_args) {
     // Doing upsteam-update
     if ( $this->isUpstreamUpdate() || $this->yamlGetSiteSetting( $name, 'upstream' ) ) {
 
+      // Set connection mode back to git.
+      if ($mode == 'sftp'||$mode == 'git') {
+        $workflow = $env->changeConnectionMode('git');
+        if (is_string($workflow)) {
+          $this->log()->info($workflow);
+        } else {
+          $workflow->wait();
+          $this->workflowOutput($workflow);
+        }
+        $mode = 'git';
+      }
+
+      // Perform upstream update
       $upstreamUpdate = $this->upstream_update( $name, 'dev', true );
 
       $proceed = $upstreamUpdate['state'];
 
+      // Set connection mode to sftp.
+      if ($mode == 'git') {
```

Comment:
If I'm not wrong, this if doesn't make much sense, since at this point you will be in git mode. You are changing it above.